decimal64.go

```diff
@@ -495,3 +517,34 @@ func propagateNan(dp, ep *decParts) *Decimal64 {
 	}
 	return ep.dec
 }
+
+// Class takes one operand and provides the class the decimal is in
```

Comment:
Is this a formally documented function in the standard? Provide a link, if available.

Reply:
this is documented in the official IEEE document under section 5.7.2 General operations, there is a html link that describes this function http://speleotrove.com/decimal/damisc.html#refclass